src/main/java/com/atlassian/db/replica/api/jdbc/JdbcUrl.java

```diff
@@ -0,0 +1,51 @@
+package com.atlassian.db.replica.api.jdbc;
+
+import java.net.URI;
+
+public class JdbcUrl {
+    private static final String PREFIX = "jdbc";
+
+    private final String internalUrl;
+
+    public JdbcUrl(JdbcProtocol protocol, String endpoint, String database) {
```

Comment:
We have a builder. The constructor doesn't have to be public.

Reply:
ok